libcloud/utils/py3.py

```diff
@@ -126,7 +126,7 @@ def tostring(node):
 
     def hexadigits(s):
         # s needs to be a byte string.
-        return [format(x, "x") for x in s]
+        return [format(x, "02x") for x in s]
```

Comment:
Ended up noticing a discrepancy between py2 and py3 here for integers `0-9`.

Reply:
Thanks.

This should indeed fix it and ensure leading ``0`` is always present.